.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -1,6 +1,6 @@
 name: Bug
-description: Report a bug
-title: "[Bug]: "
+description: Report a problem with the application
```

Comment:
Hopefully this makes things a little less jargon-y in the event that a member of the general public finds their way here and wants to report an issues they're having

Reply:
Being nitpicky so feel free to ignore. Maybe to speak to both worlds we say

"Report a bug or problem with the application"

Reply:
Updated in https://github.com/cisagov/getgov/pull/694/commits/b60e48689a1cf66b0b889310f7c25aa7a6facec0